flow/connectors/postgres/client.go

```diff
@@ -110,6 +108,9 @@ func (c *PostgresConnector) getRelIDForTable(ctx context.Context, schemaTable *u
 		 ON n.oid = c.relnamespace WHERE n.nspname=$1 AND c.relname=$2`,
 		schemaTable.Schema, schemaTable.Table).Scan(&relID)
 	if err != nil {
+		if errors.Is(err, pgx.ErrNoRows) {
+			return 0, shared.ErrTableDoesNotExist
```

Comment:
do we want to include table name in the error information?
Also maybe we should wrap the original error as well